tikv-configuration-file.md

```diff
@@ -50,6 +50,16 @@ This document only describes the parameters that are not included in command-lin
 + The size of a log file that triggers log rotation. Once the size of a log file is bigger than the specified threshold value, log files are rotated. The old log file is placed into the new file, and the new file name is the old file name with a timestamp suffix.
 + Default value: "300MB"
 
+### `memory-usage-limit`
+
++ Memory usage limit for the TiKV instance. Generally it's unnecessary to configure it explicitly, in which case it will be set to 75% of total available system memory. Considering the behavior of `block-cache.capacity`, it means 25% memory is reserved for OS page cache.
```

Comment:
```suggestion
+ Memory usage limit for the TiKV instance. Generally it's not necessary to configure it explicitly, in which case it will be set to 75% of total available system memory. Considering the behavior of `block-cache.capacity`, it means 25% memory is reserved for the OS page cache.
```